packages/neuron-wallet/src/models/transaction-size.ts

```diff
@@ -55,7 +55,8 @@ export default class TransactionSize {
   }
 
   public static outputData(data: string): number {
-    const bytes = serializeFixVec(data)
+    const fixvecCodec = fixvec(number.Uint8)
+    const bytes = byteUtils.hexify(fixvecCodec.pack(Array.from(byteUtils.bytify(data))))
```

Comment:
```suggestion
    const bytes = byteUtils.hexify(fixvecCodec.pack(data))
```

Why do we need an `Array.from` here?

Reply:
`fixvecCodec` here is an `ArrayCodec` which only accepts `ArrayLike` parameters

<img width="800" alt="image" src="https://github.com/nervosnetwork/neuron/assets/3870972/caedb4c6-2db4-4ef7-95a9-f00db91c46fc">

Reply:
https://github.com/nervosnetwork/ckb/blob/5a7efe7a0b720de79ff3761dc6e8424b8d5b22ea/util/types/schemas/blockchain.mol#L63C30-L63C30

Can we use the `BytesVec.pack` to calculate the size? I find the `outputs_data` is a `BytesVec`

Reply:
It's cool that you pointed out this, but the original code was not **exactly** serializing `outputs_data` to a `BytesVec`, instead it calculates part of `BytesVec` size by adding some `offset`: https://github.com/nervosnetwork/neuron/blob/101b7e12b5833b3f561d7eccdc253ec2fd45466f/packages/neuron-wallet/src/models/transaction-size.ts#L57-L60

A real `BytesVec` contains `header` (4 bytes), `offset` (4 bytes), `body` (4 bytes body size, followed by original data), but original code only contains `4 bytes body size` and `4 bytes offset`, which is less than result of `BytesVec.pack(data).bytelength` by 4 bytes. so the result could be like:
```
public static outputData(data: string): number {
  return BytesVec.pack([byteUtils.bytify(data)]).byteLength - 4
}
```
The code looks like magic.


We should be careful here, so maybe we can just exactly do every step as before, and use lumos. but not changing implementation here.